src/exercise/03.html

```diff
@@ -5,26 +5,26 @@
   <div id="root"></div>
   <script src="https://unpkg.com/react@17.0.0/umd/react.development.js"></script>
   <script src="https://unpkg.com/react-dom@17.0.0/umd/react-dom.development.js"></script>
-
-  <!-- 💣 remove this line
   <script src="https://unpkg.com/@babel/standalone@7.12.4/babel.js"></script>
-  💣 remove this line -->
-
-  <script type="module">
-    // 🐨 on the script tag above, change `type="module"`
-    // to `type="text/babel"` so babel will compile this code for the browser to run.
-
-    // 🐨 re-implement this using JSX!
-    const element = React.createElement('div', {
-      className: 'container',
-      children: 'Hello World',
-    })
-
-    // 💰 there are a few subtle differences between JSX and HTML. One such
-    // difference is how you apply a class to an element in JSX is by using
-    // `className` rather than `class`!
-    // 📜 You can learn the differences between JSX and HTML syntax from the React docs here:
-    // https://reactjs.org/docs/dom-elements.html#differences-in-attributes
+
+  <script type="text/babel">
+    // Set the string content
+    const greet = 'Hello';
+    const subject = 'World';
+
+    // Put together the greeting
+    const message = `${greet} ${subject}`;
+
+    // Set a class name
+    const cla = 'container';
+
+    // Make an object with the message and classname
+    const props = { msg: message, cl: cla };
+
+    // Get fancy with destructing and spread operator
+    const {msg, ...cl} = props; // could have sworn I could use new variable names here
```

Comment:
Yeah, you should be able to create new names here. You shouldn't need the spread operator.